### PR TITLE
[14.0][IMP] stock_mts_mto_rule

### PR DESCRIPTION
**Pull request by InsanivimusOmnes**
stock_mts_mto_rule: Add readme directory

### CODE REVIEW

File: stock_mts_mto_rule/__manifest__.py [hunk 1]
```diff
@@ -12,5 +12,6 @@
     "application": False,
     "installable": True,
     "depends": ["stock"],
+    "maintainers": ["InsanivimusOmnes", "CetmixGitDrone"],
```
Comment (rousseldenis) [inline stock_mts_mto_rule/__manifest__.py]: @florian-dacosta Do you maybe want to add yourself here ?

Comment (florian-dacosta) [reply stock_mts_mto_rule/__manifest__.py]: @rousseldenis I did not look in to it since a while, there is no need for me

Comment (rousseldenis) [inline stock_mts_mto_rule/__manifest__.py]: @InsanivimusOmnes Do you really want to adopt this module as I cannot see any contribution for this module apart making fragments ?

File: stock_mts_mto_rule/readme/CONTRIBUTORS.rst [hunk 2]
```diff
@@ -0,0 +1,3 @@
+* Florian da Costa <florian.dacosta@akretion.com>
+* Jared Kipe <jared@hibou.io>
+* Cetmix <https://cetmix.com>
```
Comment (rousseldenis) [inline stock_mts_mto_rule/readme/CONTRIBUTORS.rst]: This should contain only individuals.

The company should appear in the 'author' manifest key. But as said above, you haven't authored this module. So, please modify this

File: stock_mts_mto_rule/readme/CONTRIBUTORS.rst [hunk 3]
```diff
@@ -0,0 +1,2 @@
+* Florian da Costa <florian.dacosta@akretion.com>
+* Jared Kipe <jared@hibou.io>
```
Comment (rousseldenis) [inline stock_mts_mto_rule/readme/CONTRIBUTORS.rst:2]: @InsanivimusOmnes But you can add yourself here :smiley:

Comment (rousseldenis) [reply stock_mts_mto_rule/readme/CONTRIBUTORS.rst:2]: I wanted to say you as individual (but maybe your company does not want to)

**Comment by InsanivimusOmnes** [comment]
@rousseldenis please check and approve

File: README.md [hunk 4]
```diff
@@ -64,7 +64,7 @@ addon | version | maintainers | summary
 [stock_move_auto_assign](stock_move_auto_assign/) | 14.0.1.0.0 |  | Try to reserve moves when goods enter in a location
 [stock_move_common_dest](stock_move_common_dest/) | 14.0.1.0.0 |  | Adds field for common destination moves
 [stock_move_location](stock_move_location/) | 14.0.1.3.0 |  | This module allows to move all stock in a stock location to an other one.
-[stock_mts_mto_mrp_rule](stock_mts_mto_mrp_rule/) | 14.0.1.0.0 | [![geomer198](https://github.com/geomer198.png?size=30px)](https://github.com/geomer198) [![CetmixGitDrone](https://github.com/CetmixGitDrone.png?size=30px)](https://github.com/CetmixGitDrone) | Stock MTS+MTO MRP Rule
+[stock_mts_mto_mrp_rule](stock_mts_mto_mrp_rule/) | 14.0.1.0.1 | [![geomer198](https://github.com/geomer198.png?size=30px)](https://github.com/geomer198) [![CetmixGitDrone](https://github.com/CetmixGitDrone.png?size=30px)](https://github.com/CetmixGitDrone) | Stock MTS+MTO MRP Rule
```
Comment (rousseldenis) [inline README.md:67]: @InsanivimusOmnes Could you remove this ?

Comment (rousseldenis) [reply README.md:67]: Oh no, ok.

**Review by rousseldenis — APPROVED** [review]

**Pull request merged**